IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1226,24 +1117,12 @@ EOF
 sub _build_interpreter_tag_text {
     my $self = shift;
 
-    if ( $self->is_zcode ) {
-        return <<EOF;
-<script src="/static/interpreter/parchment/jquery.min.js"></script>
-<script src="/static/interpreter/parchment/parchment.min.js"></script>
-<script src="/static/interpreter/transcript_recorder/if-recorder.js"></script>
-<link rel="stylesheet" type="text/css" href="/static/interpreter/parchment/parchment.css">
-<link rel="stylesheet" type="text/css" href="/static/interpreter/parchment/style.css">
+    return <<EOF;
```

Comment:
I couldn't tell if I could just delete this whole function, but it's no longer being called from _create_parchment_page